.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,18 @@
+{
+  "image": "ghcr.io/gregac/sonata-software-container:latest",
+  "remoteUser": "sonata",
+  "containerUser": "sonata",
+  "onCreateCommand": "cd /workspaces && git clone https://github.com/lowRISC/sonata-system.git",
```

Comment:
This is cloned here as the intent is we want to be able to send out instructions to hackathon participants ASAP so they can trial this if they want, which will mean running the cmake build commands in sonata-system.

Should we have a xmake powered build available in 'sonata-software' ASAP we can drop this bit.